models/task/task.go

```diff
@@ -88,7 +88,7 @@ func (u *TaskStruct) Initialize() {
 	u.database = interfaceDB
 }
 
-func (u *TaskStruct) Save(repo *repository.RepositoryStruct, OauthToken *sql.NullString) bool {
+func (u *TaskStruct) Save(repo *repository.RepositoryStruct, OauthToken *sql.NullString) (e error) {
```

Comment:
__[golint]__
* [golint] exported method TaskStruct.Save should have comment or be unexported